### ISSUE TITLE
broken functionality between 4.9 and 6.3: cellEditing

### ISSUE THREAD

**Issue by KES777** [olifolkerd/tabulator#4627]
**Describe the bug**
In 4.9 when I subscribed to `cellEditing` callback, this callback is called when editor is ready.
In 6.3 this callback is called **before** editor is created. I see that `onRendered` event is called for the custom editor https://tabulator.info/docs/6.3/edit#edit-custom and I can do something when editor is ready.
But for builtin editors there is no such call back, eg. I can not do:
```
  editor: "list", editorParams: { onRendered: function(){ alert( 'YES' ) } }
```

I understand that the new behavior is better, but there should be a way to subscribe to `onRendered` event for built in editors.

**Tabulator Info**
- 6.3

**Working Example**
```
<div id="customized-table"></div>
<script>
new Tabulator( '#customized-table', {
  columns: [
    { editor: "list", editorParams: { onRendered: function(){ alert( 'YES' ) } } }
  ]
}
</script>
```

**To Reproduce**
1. Edit cell

**Expected behavior**
The event when **builtin** editor is ready should be called.

**Actual behavior**
I can not subscribe to callback when builtit editor is rendered.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: LInux Mint 21.3 Cinnamon
 - Browser FF
 - Version 130

**Comment by olifolkerd** [comment]
Hey @KES777 

You can always add a custom editor if you would prefer different behaviours for these. the whole point of the built in editors is to be entirely self contained, the correct approach for bespoke behaviour is either to use the new event system or use a custom editor.

Cheers

Oli :)

**Comment by KES777** [comment]
So there is no way to subscribe to `onRendered` event for built-in editors, right?

**Comment by olifolkerd** [comment]
correct, if you could explain what your end goal is i might be able to take you through the best approach.

Cheers

Oli :)

**Comment by KES777** [comment]
In the previous version when user clicks on the cell and editor with value created I used callback to select text/value inside the editor. This behavior allows user to start typing and easily replace it with new one. In the new version the user is forced to select value manually, eg. `HOME`, `SHIFT+END`. I can not pre select that value when editor is ready.
